src/components/ebay-combobox/component.ts

```diff
@@ -117,14 +118,7 @@ export default class Combobox extends Marko.Component<Input, State> {
     }
 
     handleExpand() {
-        if (this.state.viewAllOptions) {
-            this.setSelectedView();
-        } else {
-            this.state.viewAllOptions = true;
-            this.once("update", () => {
-                this.setSelectedView();
-            });
-        }
```

Comment:
I refactored this logic. Not sure If I am missing something, let me know.

Reply:
This seems like this won't work in the default usecase.

Reply:
The way I think of it now is, if `input.viewAllOptions` set to true It always shows all options whereas filtered when set to false. Curious why we want to do both?